tests/rptest/services/polaris_catalog.py

```diff
@@ -137,12 +137,14 @@ def _polaris_ready():
             self.logger.debug(
                 f"Querying polaris healthcheck on http://{node.account.hostname}:8182/healthcheck"
             )
-            r = requests.get(
-                f"http://{node.account.hostname}:8182/healthcheck", timeout=10)
 
-            self.logger.info(
-                f"health check result status code: {r.status_code}")
-            return r.status_code == 200
+            out = node.account.ssh_output(
+                "curl -s -m 10 -o /dev/null -w '%{http_code}' http://localhost:8182/healthcheck"
```

Comment:
probably good to double check with cdt, I vaguely remember `localhost` doesn't resolve on ec2 hosts (I could be wrong tho)